test/domain/notification/getAllAggregated.spec.ts

```diff
@@ -15,29 +15,45 @@ beforeEach(async () =>
 
 describe('domain/notification/getallAggregated', () =>
 {
-    it('should give all posts for the requester', async () =>
+    it('should give all posts for the requester2', async () =>
```

Comment:
As discussed before, this description should tell what scenario is tested. Getting the posts (notifications?) for requester 2 doesn't say anything. It should be clear what's wrong if this test case fails.